README.md

```diff
@@ -161,11 +161,12 @@ The instructions below assume that you are using Linux (native or WSL) or macOS.
 * CMake 3.13+
     * If you are on Ubuntu 18.04, you must first [update CMake by using the official CMake APT repository](https://apt.kitware.com/).
 * ccache (to speed up builds)
+* xdelta3
 
 Ubuntu users can install those dependencies by running:
 
 ```shell
-sudo apt install python3 ninja-build cmake ccache
+sudo apt install python3 ninja-build cmake ccache xdelta
```

Comment:
afaict the Ubuntu/Debian package is actually called xdelta3

Reply:
https://packages.ubuntu.com/search?keywords=xdelta&searchon=names&suite=focal&section=all

Yeah, I got it a little wrong since there is also an `xdelta` package (it's xdelta 1.1.3, apparently) + the macOS package manager Brew has xdelta 3 as just `xdelta`